general_qc_report/general_qc_report.py

```diff
@@ -243,7 +243,8 @@ def parse_experiment_file_qc(e, f, q, report_type, base_url, args, references_da
             for r in f.get('biological_replicates', [])
             if f.get('biological_replicates', [])
         }),
-        'has_frip': has_frip
+        'has_frip': has_frip,
+        'analysis_step': f.get('analysis_step_version').get('name') #added
```

Comment:
Looks good. Would you make this return {} though so second get won't fail out if file doesn't have analysis_step_version for some reason? 

Like:
`f.get('analysis_step_version', {}).get('name')`